spec/helpers/time_helper_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe TimeHelper do
+  describe 'date_range' do
+    context 'same day' do
+      from_date = Time.local(Time.zone.now.year, 3, 25, 12)
```

Comment:
Do not use `Time.local` without zone. Use one of `Time.zone.local`, `Time.local.current`, `Time.local.in_time_zone`, `Time.local.utc`, `Time.local.getlocal`, `Time.local.iso8601`, `Time.local.jisx0301`, `Time.local.rfc3339`, `Time.local.to_i`, `Time.local.to_f` instead.